R_program/CallSTEPWAT2.R

```diff
@@ -198,6 +278,13 @@ foreach (g = 1:length(GCM)) %dopar% { # loop through all the GCMs
               system(paste0("mv ", "sxwprod_v2.", downscaling_method, ".", y, ".", r, ".", GCM[g], ".in", " sxwprod_v2.in"))
```

Comment:
this needs to be changed to downscaling.method, which is set in Main.R here and elsewhere.